codegen/templates/endpoint.tmpl

```diff
@@ -73,10 +80,16 @@ func Handle{{title .Name}}Request(
 		return
 	}
 
+	// TODO(sindelar): Add response headers as an thrift spec annotation.
+	endRespHead := map[string]string{}
+	for _, k := range {{.Headers | printf "%#v" }} {
```

Comment:
There is no header white list here, if the workflow returns headers we should write all headers to the response.